chapters.md

```diff
@@ -3,31 +3,84 @@
 
 # Chapters
 
-## Edition and Chapter Flags
+The Matroska Chapters system can have multiple `Editions` and each `Edition` can consist of `Simple Chapters` where a chapter start time is used as marker in the timeline only. An `Edition` can be more complex with `Ordered Chapters` where a chapter end time stamp is additionally used or much more complex with `Linked Chapters`. The Matroska Chapters system can also have a menu structure, borrowed from the DVD menu system, or have it's own Native Matroska menu structure.
 
-### Chapter Flags
+## EditionEntry
```

Comment:
Here is a better description of the edition elements with samples.
https://github.com/hubblec4/Matroska-Playback/blob/master/src/EditionEntry.md

Reply:
I picked some stuff from the generale description. The rest don't seem to apply easily to this PR.

Reply:
This is OK.